apps/src/manual_registration/manual_registration.cpp

```diff
@@ -216,12 +230,30 @@ ManualRegistration::calculatePressed()
   }
   pcl::registration::TransformationEstimationSVD<pcl::PointXYZ, pcl::PointXYZ> tfe;
   tfe.estimateRigidTransformation(src_pc_, dst_pc_, transform_);
-  std::cout << "Transform : " << std::endl << transform_ << std::endl;
+  PCL_INFO("Calculated tranform:\n%1.5f %1.5f %1.5f %1.5f\n%1.5f %1.5f %1.5f "
```

Comment:
That would be better, i just copied what the original code already used.

Reply:
addressed in eea00684a4ef4637d8294ccda1f8f2d643a1b10b.

however, I have noticed that swichting to this method now subjects the output to some terminal color shenanigans, which didnt happen before. E.g. in my solarized iterm the output now looks like this

<img width="386" alt="Screenshot 2023-03-22 at 17 03 26" src="https://user-images.githubusercontent.com/11613312/226966381-55ee5482-055d-40a1-beed-45ad80ef0084.png">

Presumably because that macro has this in it `pcl::console::change_text_color(CSTR, pcl::console::ATTR, pcl::console::FG);`, while the `PCL_INFO()` and friends print without this.